lib/feedjira/logger.rb

```diff
@@ -0,0 +1,79 @@
+# Feedjira.logger
+module Feedjira
+  class << self
+    attr_writer :logger
+
+    # Provides a global logger instance for Feedjira
+    # If no logger is set by the user, a logger using DEFAULTS is provided
+    # @see Feedjira::Config::DEFAULTS
+    # @see Feedjira::Logger
+    #
+    # @return [Logger]
+    def logger
+      return @logger unless @logger.nil?
+      @logger = ::Logger.new(Feedjira::Config::DEFAULTS[:logger_io])
+      @logger.level = Feedjira::Config::DEFAULTS[:logger_level]
+      @logger
+    end
+  end
+
+  # Provide some convenience methods for logging
+  # So the Progname mustn't be passed on every call
+  # @example
+  #   Feedjira::Logger.info { "My Info Message" }
+  module Logger
```

Comment:
This is one reason. The other is, that we should add the logging Facility (in this case 'Feedjira') to all log messages. By abstracting the logger method we could achieve this without thinking of it on every log message.
I've seen this used in a lot of applications using logger.